aries-site/src/examples/templates/feedback/FeedbackAnatomy.js

```diff
@@ -0,0 +1,112 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Button,
+  Card,
+  CardBody,
+  Diagram,
+  Grid,
+  Header,
+  Heading,
+  Stack,
+  ResponsiveContext,
+} from 'grommet';
+import { Annotation } from '../../../layouts';
+import { connection } from '../../../utils';
+
+const connections = [
+  connection('1', 'layer-title'),
+  connection('2', 'layer-body'),
+  connection('3', 'layer-footer'),
+];
+
+export const FeedbackAnatomy = () => {
+  const breakpoint = useContext(ResponsiveContext);
+  let columns = ['36px', 'medium', 'auto'];
+  let rows = ['86px', '180px', '86px'];
+
+  let areas = [
+    ['annotation-1', 'layer', 'empty'],
+    ['annotation-2', 'layer', 'empty-2'],
+    ['annotation-3', 'layer', 'empty-3'],
+  ];
+
+  if (['xsmall', 'small'].includes(breakpoint)) {
+    columns = ['36px', 'flex'];
+    rows = ['60px', 'small', '60px'];
+    areas = [
+      ['annotation-1', 'layer'],
+      ['annotation-2', 'layer'],
+      ['annotation-3', 'layer'],
+    ];
+  }
+  return (
+    <Stack interactiveChild="first">
+      <Grid
+        columns={columns}
+        rows={rows}
+        areas={areas}
+        gap={{ column: 'medium' }}
+      >
+        <Annotation
+          alignSelf="center"
+          id="1"
+          target="1"
+          gridArea="annotation-1"
+        />
+        <Annotation
+          alignSelf="center"
+          id="2"
+          target="2"
+          gridArea="annotation-2"
+        />
+        <Annotation
+          alignSelf="center"
+          id="3"
+          target="3"
+          gridArea="annotation-3"
+        />
+        <LayerContent gridArea="layer" />
+      </Grid>
+      <Diagram connections={connections} />
+    </Stack>
+  );
+};
+
+const LayerContent = ({ ...rest }) => (
+  <Card
```

Comment:
I think having a container for the feedback anatomy is misleading. Feedback is container agnostic. Instead, we should treat like [PageHeader's anatomy](https://design-system.hpe.design/components/pageheader#anatomy).

Also, we need to start being disciplined in how we use Card. It should only be used [according to Card's guidance](https://design-system.hpe.design/components/card#principles).

Reply:
good point I will go back and change

Reply:
ignore anatomy when your done with your class Ill be fixing in the mean time